beancount_reds_importers/libtransactionbuilder/investments.py

```diff
@@ -169,6 +175,9 @@ def generate_trade_entry(self, ot, file, counter):
             target_acct = target_acct.format(ticker=ticker)
         else:
             target_acct = target_acct.format(ticker=self.currency)
+            metadata |= self.build_metadata(file,
```

Comment:
Why is this special cased? Shouldn't it be up next to `# Build metadata`?

Reply:
This was indented too far, but after reviewing yes it can be up next to `# Build metadata`